app/views/works/_work_header_notes.html.erb

```diff
@@ -40,6 +40,8 @@
             <%= link_to related_work.parent.title.html_safe, related_work.parent %> by <%= byline(related_work.parent) %>.
           <% elsif related_work.parent.restricted? && !logged_in? %>
             <%= ts("[Restricted Work] by") %> <%= byline(related_work.parent) %>. <%= ts("Log in to view.") %>
+          <% elsif (related_work.parent.unrevealed? ||related_work.parent.anonymous?) %>
+            <%= ts("Inspired by a work in an unrevealed collection.") %>
```

Comment:
Because the words "Inspired by" are already included on line 36, I think this is going to result in text that says, "Inspired by Inspired by a work in an unrevealed collection." 

This line probably just needs to say "a work in an unrevealed collection," much like the other lines just include the work titles.

Also, I think we might need to move this up. If the unrevealed work is restricted and you're logged out, I'm pretty sure the text is going to say, "Inspired by [Restricted Work] by CREATOR.  Log in to view." The fact that it's unrevealed should trump the fact that it's restricted.